src/main/java/com/cloudbees/jenkins/support/SupportPlugin.java

```diff
@@ -68,14 +68,7 @@
 import org.jenkinsci.remoting.RoleChecker;
 import org.kohsuke.stapler.StaplerRequest;
 
-import java.io.BufferedOutputStream;
-import java.io.File;
-import java.io.FileOutputStream;
-import java.io.FilenameFilter;
-import java.io.IOException;
-import java.io.OutputStream;
-import java.io.PrintWriter;
-import java.io.StringWriter;
+import java.io.*;
```

Comment:
you need to reconfigure your IDE to not do this. It is better to keep all imports (at least as far as you don't have more than 10 from the same package)